main.py

```diff
@@ -148,14 +148,40 @@ def FillHandWithTiles(TileQueue, PlayerTiles, MaxHandSize):
 
 
 def GetScoreForWord(Word, TileDictionary):
-    Score = 0
-    for Count in range(len(Word)):
-        Score += TileDictionary[Word[Count]]
-    if len(Word) > 7:
-        Score += 20
-    elif len(Word) > 5:
-        Score += 5
-    return Score
+  done1 = 0
+  done2 = 0
+  Score = 0
+  NewWord = [] 
+  for Count in range (len(Word)):
+    Score += TileDictionary[Word[Count]]
+  if len(Word) > 7:
+    Score += 20
+  elif len(Word) > 5:
+    Score += 5
+   for Letters in range(len(Word)):
```

Comment:
There is an extra indent here which is causing errors.
![screen shot 2017-11-11 at 20 14 12](https://user-images.githubusercontent.com/28437045/32693116-f14f5894-c71c-11e7-933a-0c92293c5efb.png)